client/websocket_client/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .client import WebsocketMessageDispatcher
+from .messages import Message
+from .utils import UNINITIALIZED, ResultEvent
+
+__all__ = ["Message", "ResultEvent", "UNINITIALIZED", "WebsocketMessageDispatcher"]
```

Comment:
```suggestion
__all__ = ("Message", "ResultEvent", "UNINITIALIZED", "WebsocketMessageDispatcher")
```
`__all__` does not need mutation